internal/server/openapi.go

```diff
@@ -335,6 +335,7 @@ func setTypeInfo(t reflect.Type, schema *openapi3.Schema) {
 	case reflect.Slice:
 		if t.Elem().Kind() == reflect.Uint8 {
 			schema.Type = "string" // []byte
```

Comment:
Is this needed now that we have the PEM instead of the byte array?

Reply:
Not needed anymore, but given that this is how `encoding/json` deals with `[]byte` it seems like it doesn't hurt too much to keep it this way in case we add other `[]byte` fields in the future.